pkgs/development/interpreters/python/mk-python-derivation.nix

```diff
@@ -115,343 +120,343 @@ let
     "build-system"
   ];
 
-in
-
-{
-  name ? "${attrs.pname}-${attrs.version}",
+  genExtraAttrs =
```

Comment:
This should probably no longer be "extra" attrs, since it's no longer an overlay.